src/Logging/Logging.Batching/BatchingLogOptions.cs

```diff
@@ -1,65 +1,49 @@
-﻿using CodeMonkeys.Logging.Configuration;
-
-using System;
+﻿using System;
 
 namespace CodeMonkeys.Logging.Batching
 {
-    public class BatchingLogOptions : LogOptions
+    public class BatchLogOptions : LogOptions
```

Comment:
I‘d like to have constants for the default values

Reply:
Done